cmd/buildah/build.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/containers/storage/pkg/archive"
+	"github.com/projectatomic/buildah/imagebuildah"
+	"github.com/urfave/cli"
+)
+
+var (
+	buildFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "quiet",
+			Usage: "refrain from announcing build instructions",
+		},
+		cli.StringFlag{
+			Name:  "registry",
+			Usage: "prefix to prepend to the image name in order to pull the image",
+			Value: DefaultRegistry,
+		},
+		cli.BoolTFlag{
+			Name:  "pull",
+			Usage: "pull the image if not present",
+		},
+		cli.BoolFlag{
+			Name:  "pull-always",
+			Usage: "pull the image, even if a version is present",
+		},
+		cli.StringFlag{
+			Name:  "signature-policy",
+			Usage: "signature policy path (not usually used)",
+		},
+		cli.StringSliceFlag{
+			Name:  "arg",
+			Usage: "`argument=value` to supply to the builder",
+		},
+		cli.StringFlag{
+			Name:  "t",
+			Usage: "`tag` to apply to the built image",
+		},
+		cli.StringSliceFlag{
+			Name:  "f",
```

Comment:
Do we want this or do we just specify Dockerfile in the command.

buildah bud Dockerfile  
Or do you want to match docker syntax totally>